### PR TITLE
fix issue 23865 - ICE on attempt to insert an non overloadable symbol…

### PR DESCRIPTION
**Pull request by ghost**
… in a set

### CODE REVIEW

File: compiler/test/compilable/test20039.d [hunk 1]
```diff
@@ -1,5 +1,5 @@
 void foo()() { }
-class bar { }
+void bar(int) { }
```
Comment (SixthDot) [inline compiler/test/compilable/test20039.d:2]: The reason this is changed is that now an error would be emitted. However I'ce checked that the issue (ICE) was also reproducible with a function (https://godbolt.org/z/Y6EqqGcac)

**Comment by thewilsonator** [comment]
ICEs should go to stable

**Comment by SixthDot** [comment]
> ICEs should go to stable

sure but do you know a technique allowing to rebase without having to change 200+ commits from `pick` to `drop` ?

**Comment by thewilsonator** [comment]
Not quite sure what you're referring to, but this is a relatively small diff. You could save the diff, and build the branch anew starting from stable and then force push to your GH.

**Comment by SixthDot** [comment]
> Not quite sure what you're referring to, but this is a relatively small diff.

I was on the bad branch, the one for issue 23686... should be good now.

**Review by thewilsonator — APPROVED** [review]

**Pull request merged**